app/models/shipit/commit.rb

```diff
@@ -92,10 +92,11 @@ def self.from_github(commit)
       author ||= Anonymous.new
       committer = User.find_or_create_committer_from_github_commit(commit)
       committer ||= Anonymous.new
+      truncated_message = truncate_message(commit.commit.message)
 
       record = new(
         sha: commit.sha,
-        message: commit.commit.message,
+        message: truncated_message,
```

Comment:
`ActiveRecord::Base.new` calls `message=`, so I think you can do the truncate in `def message=(message)` only.

Reply:
We observed that this was apparently not happening in production, such that a large commit message was causing this to blow up.
https://app.bugsnag.com/shopify/shipit/errors/5eaafd2df8753600184d823b?filters[event.since][0]=30d&filters[error.status][0]=open

That said, I will test a bit more to confirm locally. Perhaps we are mistaken about what occurred?

Reply:
Yeah, that's not supposed to happen. If you don't mind I'll try to dig that issue.

Reply:
So I've but some breakpoint and I'm 100% certain the message was truncated as expected.

I'm unsure what happened in production, but I can tell with certainty that this isn't the fix you are looking for.

If I had to guess, I would say that the bug is because MySQL size is a bytesize, whereas `String#truncate` works on a character count basis.